server/src/main/java/fr/uga/l3miage/tp4/models/EcosSessionProgrammationStepEntity.java

```diff
@@ -0,0 +1,20 @@
+package fr.uga.l3miage.tp4.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import java.time.LocalDateTime;
+
+@Entity
+public class EcosSessionProgrammationStepEntity {
+    @Id
+    private Long id;
+    private final LocalDateTime dateTime = LocalDateTime.now();
```

Comment:
Le `final` n'était pas à prendre en java, mais en métier, donc que le champs n'est pas modifiable, et donc on passe par un `@Column`
```suggestion
     @Column(updatable = false)
    private LocalDateTime dateTime;
```